MSAL/src/public/MSALSignoutParameters.h

```diff
@@ -45,6 +45,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) BOOL signoutFromBrowser;
 
+/*
+  Removes account from the keychain with either com.microsoft.adalcache shared group by default or the one provided when configuring MSALPublicClientApplication.
+
+  This is a destructive action and will remove the SSO state from all apps sharing the same cache!
+  It's intended to be used only as a way to achieve GDPR compliance and make sure all user artifacts are cleaned on user sign out.
```

Comment:
:+1:

Reply:
I guess the wipeAccount flag won't work for `externalAccountProviders`, right?

Reply:
Great catch. Let me address that one in a separate PR :)